tests/unit/jbi/whiteboard_actions/test_default.py

```diff
@@ -112,6 +112,21 @@ def test_added_comment(webhook_comment_example: BugzillaWebhookRequest, mocked_j
     assert value["status"] == "comment"
 
 
+def test_added_private_comment(
+    webhook_private_comment_example: BugzillaWebhookRequest, mocked_jira
+):
+
+    callable_object = default.init(jira_project_key="")
+
+    value = callable_object(payload=webhook_private_comment_example)
+
+    mocked_jira().issue_add_comment.assert_called_once_with(
+        issue_key="JBI-234",
+        comment="*(mathieu@mozilla.org)* commented: \n{quote}hello{quote}",
+    )
+    assert value["status"] == "comment"
+
```

Comment:
Would you mind adding a tiny test where the private comment is not found (to test the `noop`  code path)?

```py
mocked_bugzilla().get_comments.return_value = {
     "bugs": {
         str(webhook_private_comment_example.bug.id): {
            "comments": []
         }
      }
 }     
```

Reply:
Done.